### DISCUSSION TITLE
v1.1.3

### DISCUSSION THREAD

**Discussion by bghira** [bghira/SimpleTuner#1079] [Announcements]
- Nested subdir datasets will now have caches also nested in subdirectories, which unfortunately requires most-likely regenerating these entries. Sorry - it was not feasible to keep the old structure working in parallel.
- FlashAttention3 fixes for H100 nodes by downgrading default torch version to 2.4.1
- Resume fixes for multi-gpu/multi-node state/epoch tracking
- Other misc bugfixes

## What's Changed

* fix flux attn masked transformer modeling code by @bghira in https://github.com/bghira/SimpleTuner/pull/1055
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1056
* fix rope function for FA3 by @bghira in https://github.com/bghira/SimpleTuner/pull/1057
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1058
* lokr: resume by default training state if not found by @bghira in https://github.com/bghira/SimpleTuner/pull/1060
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1061
* Restore init_lokr_norm functionality by @imit8ed in https://github.com/bghira/SimpleTuner/pull/1065
* refactor how masks are retrieved by @bghira in https://github.com/bghira/SimpleTuner/pull/1066
* nvidia dependency update for pytorch-triton / aiohappyeyeballs by @bghira in https://github.com/bghira/SimpleTuner/pull/1062
* downgrade cuda to pt241 by default by @bghira in https://github.com/bghira/SimpleTuner/pull/1067
* add nightly build for pt26 by @bghira in https://github.com/bghira/SimpleTuner/pull/1068
* Add recropping script for image JSON metadata backends by @AmericanPresidentJimmyCarter in https://github.com/bghira/SimpleTuner/pull/1063
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1069
* bugfix: restore sampler state on rank 0 correctly by @bghira in https://github.com/bghira/SimpleTuner/pull/1071
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1072
* fix vae cache dir creation for subdirs by @bghira in https://github.com/bghira/SimpleTuner/pull/1076
* fix for nested image subdirs w/ duplicated filenames across subdirs by @bghira in https://github.com/bghira/SimpleTuner/pull/1078

## New Contributors
* @imit8ed made their first contribution in https://github.com/bghira/SimpleTuner/pull/1065

**Full Changelog**: https://github.com/bghira/SimpleTuner/compare/v1.1.2...v1.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/bghira/SimpleTuner/releases/tag/v1.1.3'>v1.1.3</a>.</em>
